src/Components/ProfileListPage.js

```diff
@@ -329,3 +340,18 @@ function getItemsText(items) {
 
   return `${items.length} items`;
 }
+
+function GetPrivacySymbol({ privateList }) {
```

Comment:
super nit: functions usually have names that start with verbs, but components are usually just nouns.  i.e. this could be "PrivacySymbol", then used in JSX would be `<PrivacySymbol>`, similar to other things like `<Button>` or `<Grid>`.  (unless this is the symbol for "GetPrivacy", which is an action or something).

Reply:
Great catch!! Thanks for keeping me honest.

Also, I'm sure you'll never guess what this component started as 😂😂😂